models/running_output.go

```diff
@@ -110,17 +110,17 @@ func NewRunningOutput(
 	return ro
 }
 
-func (r *RunningOutput) LogName() string {
-	return logName("outputs", r.Config.Name, r.Config.Alias)
+func (ro *RunningOutput) LogName() string {
+	return logName("outputs", ro.Config.Name, ro.Config.Alias)
```

Comment:
Does this renaming really make sense? I mean almost all plugins do have one-letter var-names for this...

Reply:
`rule.receiver-naming` requires that receiver names should be consistent (have the same var-name) and they weren't in this case. I changed all of them to `ro` (I like letter-per-word approach) but I can change them to `r` if you insist - still some changes to these receiver will have to be made...

Reply:
I'm not insisting, I'm just asking if enforcing these names makes sense? If you think it does, please change those.

Reply:
@srebhan changed to `r`

Reply:
yeah. maybe we should use one letter receivers.